src/pages/workspace/bills/WorkspaceBillsFirstSection.js

```diff
@@ -54,11 +56,23 @@ const WorkspaceBillsFirstSection = (props) => {
                 <Text>
                     {props.translate('workspace.bills.askYourVendorsBeforeEmail')}
                     {props.user.isFromPublicDomain ? (
-                        <TouchableOpacity
-                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
-                        >
-                            <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
-                        </TouchableOpacity>
+                        <>
+                            {
+                                Platform.OS === 'android' || Platform.OS === 'ios' ? (
```

Comment:
We don't follow this pattern. Use the https://github.com/Expensify/App#platform-specific-file-extensions technique for platform-specific code.